### DISCUSSION TITLE
wetransfer.com like service implementing OpenPGP.js

### DISCUSSION THREAD

**Discussion by starfrosch** [openpgpjs/openpgpjs#1224] [Q&A]
We're looking for a wetransfer.com like service that implements client side OpenPGP.js hosted and operated under Swiss Law. Any hints about a service like this? Your help is appreciated.

**Comment by steverecio** [comment]
@starfrosch I run a service that sounds like what you're looking for called Pipefile ([pipefile.com](https://pipefile.com)). Feel free to send me a note if you have any questions.

**Comment by starfrosch** [comment]
Something like this with client side encryption. https://send.tresorit.com/

**Reply by twiss** [reply]
I haven't looked at it in-depth, but the service you linked does have client side encryption:

> End-to-End Encrypted
> Your files get encrypted before sending. Content can get decrypted only by you and those who you share the link with.
